geps/gep-713/index.md

```diff
@@ -1,1157 +1,1082 @@
 
-We use the term “metaresource” to describe the class of objects that _only_ augment
-the behavior of another Kubernetes object, regardless of what they are targeting.
+Often, the semantics of a metaresource is tightly coupled to the relationships and connections a target has with other kinds of objects, typically organized in a hierarchy of nested contexts. In this sense, targeting a given resource kind may have the semantics of spanning effect across yet other objects to which the target is related.
 
-“Meta” here is used in its Greek sense of “more comprehensive”
-or “transcending”, and “resource” rather than “object” because “metaresource”
-is more pronounceable than “metaobject”. Additionally, a single word is better
-than a phrase like “wrapper object” or “wrapper resource” overall, although both
-of those terms are effectively synonymous with “metaresource”.
+##### Mergeability (“how”)
 
-A "Policy Attachment" is a metaresource that affects the fields in existing objects
-(like Gateway or Routes), or influences the configuration that's generated in an
-underlying data plane.
+Metaresources specify so-called *merge strategies* that dictate how multiple instances of the metaresource affecting a same resource (or section of a resource) should be dealt with in terms of calculating and enforcing a single set of rules specified by the metaresources.
 
-"Direct Policy Attachment" is when a Policy object references a single object _only_,
-and only modifies the fields of or the configuration associated with that object.
+The merge strategies typically include strategies for dealing with conflicting and/or missing specs, such as for applying default and/or override values on the target resources.
 
-"Inherited Policy Attachment" is when a Policy object references a single object
-_and any child objects of that object_ (according to some defined hierarchy), and
-modifies fields of the child objects, or configuration associated with the child
-objects.
+### Policies and Policy Attachment
 
-In either case, a Policy may either affect an object by controlling the value
-of one of the existing _fields_ in the `spec` of an object, or it may add
-additional fields that are _not_ in the `spec` of the object.
+Often when applying a metaresource, the intent has to do with specifying *rules* that control the behavior of the targets in action. E.g., for networking resources, the augmented behavior specified by a metaresource targeting an object can relate to conditions to let traffic flow or to block traffic at particular points of the network; in a different scenario, it can describe rules to mutate the traffic while it flows through the network at the point represented by the target object.
```

Comment:
I'm not sure why, but I find this paragraph difficult to follow. If there's a way to simplify the language here, it may help make it easier to understand.

Reply:
I can see how we may be piling up too many definitions and concepts here indeed.

Perhaps we're better off with "policy attachment" as simply the "attachment of a policy resource to its target", as more or less said.

In this case, should it be spelled lowercase then? Should we probably stop referring to this GEP/pattern as "Policy Attachment"?